backend/infrahub/services/adapters/cache/nats.py

```diff
@@ -149,3 +149,6 @@ async def set(
                 return False
         await self._get_kv(key).put(key=key, value=value.encode())
         return True
+
+    async def close_connection(self) -> None:
+        pass
```

Comment:
In general I prefer to have a docstring instead of just a `pass` in a function like this, the only reason being that a function with `pass` would count as a missed line in test coverage report where a docstring doesn't.